Newspack/Newspack/Styles/UIColor+SemanticColors.swift

```diff
@@ -75,11 +75,7 @@ extension UIColor {
     /// Text. Gray-80 (< iOS 13) and `UIColor.label` (> iOS 13)
     ///
     static var text: UIColor {
-        if #available(iOS 13, *) {
-            return .label
-        }
-
-        return .gray(.shade80)
```

Comment:
Are you supporting only iOS 13 & up now?

Reply:
Yep! iOS 13 from the get go, and I'll probably switch to iOS 14 and up before launch assuming all dependencies are compliant.